cant compile 2.5.3 on solaris 10 (SPARC)

# nice ./configure --prefix=/export/software/libresslinstall && make
checking build system type... sparc-sun-solaris2.10
checking host system type... sparc-sun-solaris2.10
checking for a BSD-compatible install... ./install-sh -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... ./install-sh -c -d
checking for gawk... no
checking for mawk... no
checking for nawk... nawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether make supports nested variables... (cached) yes
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking whether gcc understands -c and -o together... yes
checking for style of include used by make... GNU
checking dependency style of gcc... gcc3
checking for gcc option to accept ISO C99... -std=gnu99
checking for gcc -std=gnu99 option to accept ISO Standard C... (cached) -std=gnu99
checking how to print strings... printf
checking for a sed that does not truncate output... /usr/bin/sed
checking for grep that handles long lines and -e... /usr/sfw/bin/ggrep
checking for egrep... /usr/sfw/bin/ggrep -E
checking for fgrep... /usr/sfw/bin/ggrep -F
checking for ld used by gcc -std=gnu99... /usr/ccs/bin/ld
checking if the linker (/usr/ccs/bin/ld) is GNU ld... no
checking for BSD- or MS-compatible name lister (nm)... /usr/ccs/bin/nm -p
checking the name lister (/usr/ccs/bin/nm -p) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 786240
checking whether the shell understands some XSI constructs... yes
checking whether the shell understands "+="... yes
checking how to convert sparc-sun-solaris2.10 file names to sparc-sun-solaris2.10 format... func_convert_file_noop
checking how to convert sparc-sun-solaris2.10 file names to toolchain format... func_convert_file_noop
checking for /usr/ccs/bin/ld option to reload object files... -r
checking for objdump... no
checking how to recognize dependent libraries... pass_all
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for ar... ar
checking for archiver @FILE support... no
checking for strip... strip
checking for ranlib... ranlib
checking command to parse /usr/ccs/bin/nm -p output from gcc -std=gnu99 object... ok
checking for sysroot... no
checking for mt... mt
checking if mt is a manifest tool... no
checking how to run the C preprocessor... gcc -std=gnu99 -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if gcc -std=gnu99 supports -fno-rtti -fno-exceptions... no
checking for gcc -std=gnu99 option to produce PIC... -fPIC -DPIC
checking if gcc -std=gnu99 PIC flag -fPIC -DPIC works... yes
checking if gcc -std=gnu99 static flag -static works... no
checking if gcc -std=gnu99 supports -c -o file.o... yes
checking if gcc -std=gnu99 supports -c -o file.o... (cached) yes
checking whether the gcc -std=gnu99 linker (/usr/ccs/bin/ld) supports shared libraries... yes
checking whether -lc should be explicitly linked in... yes
checking dynamic linker characteristics... solaris2.10 ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... no
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... yes
checking if gcc -std=gnu99 supports "-fno-strict-overflow"... no
checking if gcc -std=gnu99 supports "-D_FORTIFY_SOURCE=2"... no
checking if /usr/ccs/bin/ld supports "-Wl,-z,relro"... no
checking if /usr/ccs/bin/ld supports "-Wl,-z,now"... no
checking if gcc -std=gnu99 supports "-fstack-protector-strong"... no
checking if gcc -std=gnu99 supports "-fstack-protector-all"... no
configure: WARNING: compiler does not appear to support stack protection
checking whether AS supports .note.GNU-stack... no
checking dependency style of gcc -std=gnu99... gcc3
checking if compiling with clang... no
checking whether CC supports -Wno-pointer-sign... no
checking for symlink... yes
checking for funopen... no
checking err.h usability... no
checking err.h presence... no
checking for err.h... no
checking readpassphrase.h usability... no
checking readpassphrase.h presence... no
checking for readpassphrase.h... no
checking for asprintf... yes
checking for getpagesize... yes
checking for inet_ntop... no
checking for inet_pton... no
checking for memmem... no
checking for readpassphrase... no
checking for reallocarray... no
checking for recallocarray... no
checking for strlcat... yes
checking for strlcpy... yes
checking for strndup... no
checking for strnlen... no
checking for strsep... no
checking for strtonum... no
checking for timegm... no
checking for _mkgmtime... no
checking for accept4... no
checking for pipe2... no
checking for pledge... no
checking for poll... yes
checking for socketpair... no
checking for arc4random... no
checking for arc4random_buf... no
checking for arc4random_uniform... no
checking for explicit_bzero... no
checking for getauxval... no
checking for getentropy... no
checking for timingsafe_bcmp... no
checking for timingsafe_memcmp... no
checking for getauxval... (cached) no
checking for library containing clock_gettime... -lrt
checking for clock_gettime... yes
checking for library containing dl_iterate_phdr... -ldl
checking for dl_iterate_phdr... yes
checking whether va_copy exists... yes
checking whether __va_copy exists... yes
checking for library containing b64_ntop... -lresolv
checking for library containing __b64_ntop... none required
checking for b64_ntop... yes
generating ./crypto/crypto_portable.sym ...
checking if BSWAP4 builds without __STRICT_ALIGNMENT... no
checking if .gnu.warning accepts long strings... no
checking size of time_t... 4
 ** Warning, this system is unable to represent times past 2038
 ** It will behave incorrectly when handling valid RFC5280 dates
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating include/Makefile
config.status: creating include/openssl/Makefile
config.status: creating crypto/Makefile
config.status: creating ssl/Makefile
config.status: creating tls/Makefile
config.status: creating tests/Makefile
config.status: creating apps/Makefile
config.status: creating apps/ocspcheck/Makefile
config.status: creating apps/openssl/Makefile
config.status: creating apps/nc/Makefile
config.status: creating man/Makefile
config.status: creating libcrypto.pc
config.status: creating libssl.pc
config.status: creating libtls.pc
config.status: creating openssl.pc
config.status: executing depfiles commands
config.status: executing libtool commands
Making all in crypto
  CC       aes/libcrypto_la-aes_cbc.lo
  CC       aes/libcrypto_la-aes_core.lo
  CC       camellia/libcrypto_la-camellia.lo
  CC       camellia/libcrypto_la-cmll_cbc.lo
  CC       rc4/libcrypto_la-rc4_enc.lo
  CC       rc4/libcrypto_la-rc4_skey.lo
  CC       whrlpool/libcrypto_la-wp_block.lo
  CC       libcrypto_la-cpt_err.lo
  CC       libcrypto_la-cryptlib.lo
  CC       libcrypto_la-cversion.lo
  CC       libcrypto_la-ex_data.lo
  CC       libcrypto_la-malloc-wrapper.lo
  CC       libcrypto_la-mem_clr.lo
  CC       libcrypto_la-mem_dbg.lo
  CC       libcrypto_la-o_init.lo
  CC       libcrypto_la-o_str.lo
  CC       libcrypto_la-o_time.lo
  CC       aes/libcrypto_la-aes_cfb.lo
  CC       aes/libcrypto_la-aes_ctr.lo
  CC       aes/libcrypto_la-aes_ecb.lo
  CC       aes/libcrypto_la-aes_ige.lo
  CC       aes/libcrypto_la-aes_misc.lo
  CC       aes/libcrypto_la-aes_ofb.lo
  CC       aes/libcrypto_la-aes_wrap.lo
  CC       asn1/libcrypto_la-a_bitstr.lo
  CC       asn1/libcrypto_la-a_bool.lo
  CC       asn1/libcrypto_la-a_bytes.lo
  CC       asn1/libcrypto_la-a_d2i_fp.lo
  CC       asn1/libcrypto_la-a_digest.lo
  CC       asn1/libcrypto_la-a_dup.lo
  CC       asn1/libcrypto_la-a_enum.lo
  CC       asn1/libcrypto_la-a_i2d_fp.lo
  CC       asn1/libcrypto_la-a_int.lo
  CC       asn1/libcrypto_la-a_mbstr.lo
  CC       asn1/libcrypto_la-a_object.lo
  CC       asn1/libcrypto_la-a_octet.lo
  CC       asn1/libcrypto_la-a_print.lo
  CC       asn1/libcrypto_la-a_set.lo
  CC       asn1/libcrypto_la-a_sign.lo
  CC       asn1/libcrypto_la-a_strex.lo
  CC       asn1/libcrypto_la-a_strnid.lo
  CC       asn1/libcrypto_la-a_time.lo
  CC       asn1/libcrypto_la-a_time_tm.lo
asn1/a_time_tm.c: In function `gentime_string_from_tm':
asn1/a_time_tm.c:72: warning: implicit declaration of function `asprintf'
  CC       asn1/libcrypto_la-a_type.lo
  CC       asn1/libcrypto_la-a_utf8.lo
  CC       asn1/libcrypto_la-a_verify.lo
  CC       asn1/libcrypto_la-ameth_lib.lo
  CC       asn1/libcrypto_la-asn1_err.lo
  CC       asn1/libcrypto_la-asn1_gen.lo
  CC       asn1/libcrypto_la-asn1_lib.lo
  CC       asn1/libcrypto_la-asn1_par.lo
  CC       asn1/libcrypto_la-asn_mime.lo
  CC       asn1/libcrypto_la-asn_moid.lo
  CC       asn1/libcrypto_la-asn_pack.lo
  CC       asn1/libcrypto_la-bio_asn1.lo
  CC       asn1/libcrypto_la-bio_ndef.lo
  CC       asn1/libcrypto_la-d2i_pr.lo
  CC       asn1/libcrypto_la-d2i_pu.lo
  CC       asn1/libcrypto_la-evp_asn1.lo
  CC       asn1/libcrypto_la-f_enum.lo
  CC       asn1/libcrypto_la-f_int.lo
  CC       asn1/libcrypto_la-f_string.lo
  CC       asn1/libcrypto_la-i2d_pr.lo
  CC       asn1/libcrypto_la-i2d_pu.lo
  CC       asn1/libcrypto_la-n_pkey.lo
  CC       asn1/libcrypto_la-nsseq.lo
  CC       asn1/libcrypto_la-p5_pbe.lo
  CC       asn1/libcrypto_la-p5_pbev2.lo
  CC       asn1/libcrypto_la-p8_pkey.lo
  CC       asn1/libcrypto_la-t_bitst.lo
  CC       asn1/libcrypto_la-t_crl.lo
  CC       asn1/libcrypto_la-t_pkey.lo
  CC       asn1/libcrypto_la-t_req.lo
  CC       asn1/libcrypto_la-t_spki.lo
  CC       asn1/libcrypto_la-t_x509.lo
  CC       asn1/libcrypto_la-t_x509a.lo
  CC       asn1/libcrypto_la-tasn_dec.lo
  CC       asn1/libcrypto_la-tasn_enc.lo
  CC       asn1/libcrypto_la-tasn_fre.lo
  CC       asn1/libcrypto_la-tasn_new.lo
  CC       asn1/libcrypto_la-tasn_prn.lo
  CC       asn1/libcrypto_la-tasn_typ.lo
  CC       asn1/libcrypto_la-tasn_utl.lo
  CC       asn1/libcrypto_la-x_algor.lo
  CC       asn1/libcrypto_la-x_attrib.lo
  CC       asn1/libcrypto_la-x_bignum.lo
  CC       asn1/libcrypto_la-x_crl.lo
  CC       asn1/libcrypto_la-x_exten.lo
  CC       asn1/libcrypto_la-x_info.lo
  CC       asn1/libcrypto_la-x_long.lo
  CC       asn1/libcrypto_la-x_name.lo
  CC       asn1/libcrypto_la-x_nx509.lo
  CC       asn1/libcrypto_la-x_pkey.lo
  CC       asn1/libcrypto_la-x_pubkey.lo
  CC       asn1/libcrypto_la-x_req.lo
  CC       asn1/libcrypto_la-x_sig.lo
  CC       asn1/libcrypto_la-x_spki.lo
  CC       asn1/libcrypto_la-x_val.lo
  CC       asn1/libcrypto_la-x_x509.lo
  CC       asn1/libcrypto_la-x_x509a.lo
  CC       bf/libcrypto_la-bf_cfb64.lo
  CC       bf/libcrypto_la-bf_ecb.lo
  CC       bf/libcrypto_la-bf_enc.lo
  CC       bf/libcrypto_la-bf_ofb64.lo
  CC       bf/libcrypto_la-bf_skey.lo
  CC       bio/libcrypto_la-b_dump.lo
  CC       bio/libcrypto_la-b_posix.lo
  CC       bio/libcrypto_la-b_print.lo
bio/b_print.c: In function `BIO_vprintf':
bio/b_print.c:51: warning: implicit declaration of function `vasprintf'
  CC       bio/libcrypto_la-b_sock.lo
  CC       bio/libcrypto_la-bf_buff.lo
  CC       bio/libcrypto_la-bf_nbio.lo
  CC       bio/libcrypto_la-bf_null.lo
  CC       bio/libcrypto_la-bio_cb.lo
  CC       bio/libcrypto_la-bio_err.lo
  CC       bio/libcrypto_la-bio_lib.lo
  CC       bio/libcrypto_la-bss_acpt.lo
  CC       bio/libcrypto_la-bss_bio.lo
  CC       bio/libcrypto_la-bss_conn.lo
bio/bss_conn.c: In function `conn_ctrl':
bio/bss_conn.c:477: warning: implicit declaration of function `asprintf'
  CC       bio/libcrypto_la-bss_dgram.lo
  CC       bio/libcrypto_la-bss_fd.lo
  CC       bio/libcrypto_la-bss_file.lo
  CC       bio/libcrypto_la-bss_log.lo
  CC       bio/libcrypto_la-bss_mem.lo
  CC       bio/libcrypto_la-bss_null.lo
  CC       bio/libcrypto_la-bss_sock.lo
  CC       bn/libcrypto_la-bn_add.lo
  CC       bn/libcrypto_la-bn_asm.lo
  CC       bn/libcrypto_la-bn_blind.lo
  CC       bn/libcrypto_la-bn_const.lo
  CC       bn/libcrypto_la-bn_ctx.lo
  CC       bn/libcrypto_la-bn_depr.lo
  CC       bn/libcrypto_la-bn_div.lo
  CC       bn/libcrypto_la-bn_err.lo
  CC       bn/libcrypto_la-bn_exp.lo
  CC       bn/libcrypto_la-bn_exp2.lo
  CC       bn/libcrypto_la-bn_gcd.lo
  CC       bn/libcrypto_la-bn_gf2m.lo
  CC       bn/libcrypto_la-bn_kron.lo
  CC       bn/libcrypto_la-bn_lib.lo
  CC       bn/libcrypto_la-bn_mod.lo
  CC       bn/libcrypto_la-bn_mont.lo
  CC       bn/libcrypto_la-bn_mpi.lo
  CC       bn/libcrypto_la-bn_mul.lo
  CC       bn/libcrypto_la-bn_nist.lo
  CC       bn/libcrypto_la-bn_prime.lo
  CC       bn/libcrypto_la-bn_print.lo
  CC       bn/libcrypto_la-bn_rand.lo
  CC       bn/libcrypto_la-bn_recp.lo
  CC       bn/libcrypto_la-bn_shift.lo
  CC       bn/libcrypto_la-bn_sqr.lo
  CC       bn/libcrypto_la-bn_sqrt.lo
  CC       bn/libcrypto_la-bn_word.lo
  CC       bn/libcrypto_la-bn_x931p.lo
  CC       buffer/libcrypto_la-buf_err.lo
  CC       buffer/libcrypto_la-buf_str.lo
  CC       buffer/libcrypto_la-buffer.lo
  CC       camellia/libcrypto_la-cmll_cfb.lo
  CC       camellia/libcrypto_la-cmll_ctr.lo
  CC       camellia/libcrypto_la-cmll_ecb.lo
  CC       camellia/libcrypto_la-cmll_misc.lo
  CC       camellia/libcrypto_la-cmll_ofb.lo
  CC       cast/libcrypto_la-c_cfb64.lo
  CC       cast/libcrypto_la-c_ecb.lo
  CC       cast/libcrypto_la-c_enc.lo
  CC       cast/libcrypto_la-c_ofb64.lo
  CC       cast/libcrypto_la-c_skey.lo
  CC       chacha/libcrypto_la-chacha.lo
  CC       cmac/libcrypto_la-cm_ameth.lo
  CC       cmac/libcrypto_la-cm_pmeth.lo
  CC       cmac/libcrypto_la-cmac.lo
  CC       comp/libcrypto_la-c_rle.lo
  CC       comp/libcrypto_la-c_zlib.lo
  CC       comp/libcrypto_la-comp_err.lo
  CC       comp/libcrypto_la-comp_lib.lo
  CC       conf/libcrypto_la-conf_api.lo
  CC       conf/libcrypto_la-conf_def.lo
  CC       conf/libcrypto_la-conf_err.lo
  CC       conf/libcrypto_la-conf_lib.lo
  CC       conf/libcrypto_la-conf_mall.lo
  CC       conf/libcrypto_la-conf_mod.lo
conf/conf_mod.c: In function `CONF_get1_default_config_file':
conf/conf_mod.c:548: warning: implicit declaration of function `asprintf'
  CC       conf/libcrypto_la-conf_sap.lo
  CC       curve25519/libcrypto_la-curve25519-generic.lo
  CC       curve25519/libcrypto_la-curve25519.lo
  CC       des/libcrypto_la-cbc_cksm.lo
  CC       des/libcrypto_la-cbc_enc.lo
  CC       des/libcrypto_la-cfb64ede.lo
  CC       des/libcrypto_la-cfb64enc.lo
  CC       des/libcrypto_la-cfb_enc.lo
  CC       des/libcrypto_la-des_enc.lo
  CC       des/libcrypto_la-ecb3_enc.lo
  CC       des/libcrypto_la-ecb_enc.lo
  CC       des/libcrypto_la-ede_cbcm_enc.lo
  CC       des/libcrypto_la-enc_read.lo
  CC       des/libcrypto_la-enc_writ.lo
  CC       des/libcrypto_la-fcrypt.lo
  CC       des/libcrypto_la-fcrypt_b.lo
  CC       des/libcrypto_la-ofb64ede.lo
  CC       des/libcrypto_la-ofb64enc.lo
  CC       des/libcrypto_la-ofb_enc.lo
  CC       des/libcrypto_la-pcbc_enc.lo
  CC       des/libcrypto_la-qud_cksm.lo
  CC       des/libcrypto_la-rand_key.lo
  CC       des/libcrypto_la-set_key.lo
  CC       des/libcrypto_la-str2key.lo
  CC       des/libcrypto_la-xcbc_enc.lo
  CC       dh/libcrypto_la-dh_ameth.lo
  CC       dh/libcrypto_la-dh_asn1.lo
  CC       dh/libcrypto_la-dh_check.lo
  CC       dh/libcrypto_la-dh_depr.lo
  CC       dh/libcrypto_la-dh_err.lo
  CC       dh/libcrypto_la-dh_gen.lo
  CC       dh/libcrypto_la-dh_key.lo
  CC       dh/libcrypto_la-dh_lib.lo
  CC       dh/libcrypto_la-dh_pmeth.lo
  CC       dh/libcrypto_la-dh_prn.lo
  CC       dsa/libcrypto_la-dsa_ameth.lo
  CC       dsa/libcrypto_la-dsa_asn1.lo
  CC       dsa/libcrypto_la-dsa_depr.lo
  CC       dsa/libcrypto_la-dsa_err.lo
  CC       dsa/libcrypto_la-dsa_gen.lo
  CC       dsa/libcrypto_la-dsa_key.lo
  CC       dsa/libcrypto_la-dsa_lib.lo
  CC       dsa/libcrypto_la-dsa_ossl.lo
  CC       dsa/libcrypto_la-dsa_pmeth.lo
  CC       dsa/libcrypto_la-dsa_prn.lo
  CC       dsa/libcrypto_la-dsa_sign.lo
  CC       dsa/libcrypto_la-dsa_vrf.lo
  CC       dso/libcrypto_la-dso_dlfcn.lo
  CC       dso/libcrypto_la-dso_err.lo
  CC       dso/libcrypto_la-dso_lib.lo
  CC       dso/libcrypto_la-dso_null.lo
  CC       dso/libcrypto_la-dso_openssl.lo
  CC       ec/libcrypto_la-ec2_mult.lo
  CC       ec/libcrypto_la-ec2_oct.lo
  CC       ec/libcrypto_la-ec2_smpl.lo
  CC       ec/libcrypto_la-ec_ameth.lo
  CC       ec/libcrypto_la-ec_asn1.lo
  CC       ec/libcrypto_la-ec_check.lo
  CC       ec/libcrypto_la-ec_curve.lo
  CC       ec/libcrypto_la-ec_cvt.lo
  CC       ec/libcrypto_la-ec_err.lo
  CC       ec/libcrypto_la-ec_key.lo
  CC       ec/libcrypto_la-ec_lib.lo
  CC       ec/libcrypto_la-ec_mult.lo
  CC       ec/libcrypto_la-ec_oct.lo
  CC       ec/libcrypto_la-ec_pmeth.lo
  CC       ec/libcrypto_la-ec_print.lo
  CC       ec/libcrypto_la-eck_prn.lo
  CC       ec/libcrypto_la-ecp_mont.lo
  CC       ec/libcrypto_la-ecp_nist.lo
  CC       ec/libcrypto_la-ecp_nistp224.lo
  CC       ec/libcrypto_la-ecp_nistp256.lo
  CC       ec/libcrypto_la-ecp_nistp521.lo
  CC       ec/libcrypto_la-ecp_nistputil.lo
  CC       ec/libcrypto_la-ecp_oct.lo
  CC       ec/libcrypto_la-ecp_smpl.lo
  CC       ecdh/libcrypto_la-ech_err.lo
  CC       ecdh/libcrypto_la-ech_key.lo
  CC       ecdh/libcrypto_la-ech_lib.lo
  CC       ecdsa/libcrypto_la-ecs_asn1.lo
  CC       ecdsa/libcrypto_la-ecs_err.lo
  CC       ecdsa/libcrypto_la-ecs_lib.lo
  CC       ecdsa/libcrypto_la-ecs_ossl.lo
  CC       ecdsa/libcrypto_la-ecs_sign.lo
  CC       ecdsa/libcrypto_la-ecs_vrf.lo
  CC       engine/libcrypto_la-eng_all.lo
  CC       engine/libcrypto_la-eng_cnf.lo
  CC       engine/libcrypto_la-eng_ctrl.lo
  CC       engine/libcrypto_la-eng_dyn.lo
  CC       engine/libcrypto_la-eng_err.lo
  CC       engine/libcrypto_la-eng_fat.lo
  CC       engine/libcrypto_la-eng_init.lo
  CC       engine/libcrypto_la-eng_lib.lo
  CC       engine/libcrypto_la-eng_list.lo
  CC       engine/libcrypto_la-eng_openssl.lo
  CC       engine/libcrypto_la-eng_pkey.lo
  CC       engine/libcrypto_la-eng_table.lo
  CC       engine/libcrypto_la-tb_asnmth.lo
  CC       engine/libcrypto_la-tb_cipher.lo
  CC       engine/libcrypto_la-tb_dh.lo
  CC       engine/libcrypto_la-tb_digest.lo
  CC       engine/libcrypto_la-tb_dsa.lo
  CC       engine/libcrypto_la-tb_ecdh.lo
  CC       engine/libcrypto_la-tb_ecdsa.lo
  CC       engine/libcrypto_la-tb_pkmeth.lo
  CC       engine/libcrypto_la-tb_rand.lo
  CC       engine/libcrypto_la-tb_rsa.lo
  CC       engine/libcrypto_la-tb_store.lo
  CC       err/libcrypto_la-err.lo
err/err.c: In function `ERR_asprintf_error_data':
err/err.c:1088: warning: implicit declaration of function `vasprintf'
  CC       err/libcrypto_la-err_all.lo
  CC       err/libcrypto_la-err_prn.lo
  CC       evp/libcrypto_la-bio_b64.lo
  CC       evp/libcrypto_la-bio_enc.lo
  CC       evp/libcrypto_la-bio_md.lo
  CC       evp/libcrypto_la-c_all.lo
  CC       evp/libcrypto_la-digest.lo
  CC       evp/libcrypto_la-e_aes.lo
  CC       evp/libcrypto_la-e_aes_cbc_hmac_sha1.lo
  CC       evp/libcrypto_la-e_bf.lo
  CC       evp/libcrypto_la-e_camellia.lo
  CC       evp/libcrypto_la-e_cast.lo
  CC       evp/libcrypto_la-e_chacha.lo
  CC       evp/libcrypto_la-e_chacha20poly1305.lo
  CC       evp/libcrypto_la-e_des.lo
  CC       evp/libcrypto_la-e_des3.lo
  CC       evp/libcrypto_la-e_gost2814789.lo
  CC       evp/libcrypto_la-e_idea.lo
  CC       evp/libcrypto_la-e_null.lo
  CC       evp/libcrypto_la-e_old.lo
  CC       evp/libcrypto_la-e_rc2.lo
  CC       evp/libcrypto_la-e_rc4.lo
  CC       evp/libcrypto_la-e_rc4_hmac_md5.lo
  CC       evp/libcrypto_la-e_xcbc_d.lo
  CC       evp/libcrypto_la-encode.lo
  CC       evp/libcrypto_la-evp_aead.lo
  CC       evp/libcrypto_la-evp_enc.lo
  CC       evp/libcrypto_la-evp_err.lo
  CC       evp/libcrypto_la-evp_key.lo
  CC       evp/libcrypto_la-evp_lib.lo
  CC       evp/libcrypto_la-evp_pbe.lo
  CC       evp/libcrypto_la-evp_pkey.lo
  CC       evp/libcrypto_la-m_dss.lo
  CC       evp/libcrypto_la-m_dss1.lo
  CC       evp/libcrypto_la-m_ecdsa.lo
  CC       evp/libcrypto_la-m_gost2814789.lo
  CC       evp/libcrypto_la-m_gostr341194.lo
  CC       evp/libcrypto_la-m_md4.lo
  CC       evp/libcrypto_la-m_md5.lo
  CC       evp/libcrypto_la-m_md5_sha1.lo
  CC       evp/libcrypto_la-m_null.lo
  CC       evp/libcrypto_la-m_ripemd.lo
  CC       evp/libcrypto_la-m_sha1.lo
  CC       evp/libcrypto_la-m_sigver.lo
  CC       evp/libcrypto_la-m_streebog.lo
  CC       evp/libcrypto_la-m_wp.lo
  CC       evp/libcrypto_la-names.lo
  CC       evp/libcrypto_la-p5_crpt.lo
  CC       evp/libcrypto_la-p5_crpt2.lo
  CC       evp/libcrypto_la-p_dec.lo
  CC       evp/libcrypto_la-p_enc.lo
  CC       evp/libcrypto_la-p_lib.lo
  CC       evp/libcrypto_la-p_open.lo
  CC       evp/libcrypto_la-p_seal.lo
  CC       evp/libcrypto_la-p_sign.lo
  CC       evp/libcrypto_la-p_verify.lo
  CC       evp/libcrypto_la-pmeth_fn.lo
  CC       evp/libcrypto_la-pmeth_gn.lo
  CC       evp/libcrypto_la-pmeth_lib.lo
  CC       gost/libcrypto_la-gost2814789.lo
  CC       gost/libcrypto_la-gost89_keywrap.lo
  CC       gost/libcrypto_la-gost89_params.lo
  CC       gost/libcrypto_la-gost89imit_ameth.lo
  CC       gost/libcrypto_la-gost89imit_pmeth.lo
  CC       gost/libcrypto_la-gost_asn1.lo
  CC       gost/libcrypto_la-gost_err.lo
  CC       gost/libcrypto_la-gostr341001.lo
  CC       gost/libcrypto_la-gostr341001_ameth.lo
  CC       gost/libcrypto_la-gostr341001_key.lo
  CC       gost/libcrypto_la-gostr341001_params.lo
  CC       gost/libcrypto_la-gostr341001_pmeth.lo
  CC       gost/libcrypto_la-gostr341194.lo
  CC       gost/libcrypto_la-streebog.lo
  CC       hmac/libcrypto_la-hm_ameth.lo
  CC       hmac/libcrypto_la-hm_pmeth.lo
  CC       hmac/libcrypto_la-hmac.lo
  CC       idea/libcrypto_la-i_cbc.lo
  CC       idea/libcrypto_la-i_cfb64.lo
  CC       idea/libcrypto_la-i_ecb.lo
  CC       idea/libcrypto_la-i_ofb64.lo
  CC       idea/libcrypto_la-i_skey.lo
  CC       lhash/libcrypto_la-lh_stats.lo
  CC       lhash/libcrypto_la-lhash.lo
  CC       md4/libcrypto_la-md4_dgst.lo
  CC       md4/libcrypto_la-md4_one.lo
  CC       md5/libcrypto_la-md5_dgst.lo
  CC       md5/libcrypto_la-md5_one.lo
  CC       modes/libcrypto_la-cbc128.lo
  CC       modes/libcrypto_la-ccm128.lo
  CC       modes/libcrypto_la-cfb128.lo
  CC       modes/libcrypto_la-ctr128.lo
  CC       modes/libcrypto_la-cts128.lo
  CC       modes/libcrypto_la-gcm128.lo
  CC       modes/libcrypto_la-ofb128.lo
  CC       modes/libcrypto_la-xts128.lo
  CC       objects/libcrypto_la-o_names.lo
  CC       objects/libcrypto_la-obj_dat.lo
  CC       objects/libcrypto_la-obj_err.lo
  CC       objects/libcrypto_la-obj_lib.lo
  CC       objects/libcrypto_la-obj_xref.lo
  CC       ocsp/libcrypto_la-ocsp_asn.lo
  CC       ocsp/libcrypto_la-ocsp_cl.lo
  CC       ocsp/libcrypto_la-ocsp_err.lo
  CC       ocsp/libcrypto_la-ocsp_ext.lo
  CC       ocsp/libcrypto_la-ocsp_ht.lo
  CC       ocsp/libcrypto_la-ocsp_lib.lo
  CC       ocsp/libcrypto_la-ocsp_prn.lo
  CC       ocsp/libcrypto_la-ocsp_srv.lo
  CC       ocsp/libcrypto_la-ocsp_vfy.lo
  CC       pem/libcrypto_la-pem_all.lo
  CC       pem/libcrypto_la-pem_err.lo
  CC       pem/libcrypto_la-pem_info.lo
  CC       pem/libcrypto_la-pem_lib.lo
  CC       pem/libcrypto_la-pem_oth.lo
  CC       pem/libcrypto_la-pem_pk8.lo
  CC       pem/libcrypto_la-pem_pkey.lo
  CC       pem/libcrypto_la-pem_seal.lo
  CC       pem/libcrypto_la-pem_sign.lo
  CC       pem/libcrypto_la-pem_x509.lo
  CC       pem/libcrypto_la-pem_xaux.lo
  CC       pem/libcrypto_la-pvkfmt.lo
  CC       pkcs12/libcrypto_la-p12_add.lo
  CC       pkcs12/libcrypto_la-p12_asn.lo
  CC       pkcs12/libcrypto_la-p12_attr.lo
  CC       pkcs12/libcrypto_la-p12_crpt.lo
  CC       pkcs12/libcrypto_la-p12_crt.lo
  CC       pkcs12/libcrypto_la-p12_decr.lo
  CC       pkcs12/libcrypto_la-p12_init.lo
  CC       pkcs12/libcrypto_la-p12_key.lo
  CC       pkcs12/libcrypto_la-p12_kiss.lo
  CC       pkcs12/libcrypto_la-p12_mutl.lo
  CC       pkcs12/libcrypto_la-p12_npas.lo
  CC       pkcs12/libcrypto_la-p12_p8d.lo
  CC       pkcs12/libcrypto_la-p12_p8e.lo
  CC       pkcs12/libcrypto_la-p12_utl.lo
  CC       pkcs12/libcrypto_la-pk12err.lo
  CC       pkcs7/libcrypto_la-bio_pk7.lo
  CC       pkcs7/libcrypto_la-pk7_asn1.lo
  CC       pkcs7/libcrypto_la-pk7_attr.lo
  CC       pkcs7/libcrypto_la-pk7_doit.lo
  CC       pkcs7/libcrypto_la-pk7_lib.lo
  CC       pkcs7/libcrypto_la-pk7_mime.lo
  CC       pkcs7/libcrypto_la-pk7_smime.lo
  CC       pkcs7/libcrypto_la-pkcs7err.lo
  CC       poly1305/libcrypto_la-poly1305.lo
  CC       rand/libcrypto_la-rand_err.lo
  CC       rand/libcrypto_la-rand_lib.lo
  CC       rand/libcrypto_la-randfile.lo
  CC       rc2/libcrypto_la-rc2_cbc.lo
  CC       rc2/libcrypto_la-rc2_ecb.lo
  CC       rc2/libcrypto_la-rc2_skey.lo
  CC       rc2/libcrypto_la-rc2cfb64.lo
  CC       rc2/libcrypto_la-rc2ofb64.lo
  CC       ripemd/libcrypto_la-rmd_dgst.lo
  CC       ripemd/libcrypto_la-rmd_one.lo
  CC       rsa/libcrypto_la-rsa_ameth.lo
  CC       rsa/libcrypto_la-rsa_asn1.lo
  CC       rsa/libcrypto_la-rsa_chk.lo
  CC       rsa/libcrypto_la-rsa_crpt.lo
  CC       rsa/libcrypto_la-rsa_depr.lo
  CC       rsa/libcrypto_la-rsa_eay.lo
  CC       rsa/libcrypto_la-rsa_err.lo
  CC       rsa/libcrypto_la-rsa_gen.lo
  CC       rsa/libcrypto_la-rsa_lib.lo
  CC       rsa/libcrypto_la-rsa_none.lo
  CC       rsa/libcrypto_la-rsa_oaep.lo
  CC       rsa/libcrypto_la-rsa_pk1.lo
  CC       rsa/libcrypto_la-rsa_pmeth.lo
  CC       rsa/libcrypto_la-rsa_prn.lo
  CC       rsa/libcrypto_la-rsa_pss.lo
  CC       rsa/libcrypto_la-rsa_saos.lo
  CC       rsa/libcrypto_la-rsa_sign.lo
  CC       rsa/libcrypto_la-rsa_ssl.lo
  CC       rsa/libcrypto_la-rsa_x931.lo
  CC       sha/libcrypto_la-sha1_one.lo
  CC       sha/libcrypto_la-sha1dgst.lo
  CC       sha/libcrypto_la-sha256.lo
  CC       sha/libcrypto_la-sha512.lo
  CC       stack/libcrypto_la-stack.lo
  CC       ts/libcrypto_la-ts_asn1.lo
  CC       ts/libcrypto_la-ts_conf.lo
  CC       ts/libcrypto_la-ts_err.lo
  CC       ts/libcrypto_la-ts_lib.lo
  CC       ts/libcrypto_la-ts_req_print.lo
  CC       ts/libcrypto_la-ts_req_utils.lo
  CC       ts/libcrypto_la-ts_rsp_print.lo
  CC       ts/libcrypto_la-ts_rsp_sign.lo
  CC       ts/libcrypto_la-ts_rsp_utils.lo
  CC       ts/libcrypto_la-ts_rsp_verify.lo
  CC       ts/libcrypto_la-ts_verify_ctx.lo
  CC       txt_db/libcrypto_la-txt_db.lo
  CC       ui/libcrypto_la-ui_err.lo
  CC       ui/libcrypto_la-ui_lib.lo
ui/ui_lib.c: In function `UI_construct_prompt':
ui/ui_lib.c:395: warning: implicit declaration of function `asprintf'
  CC       ui/libcrypto_la-ui_openssl.lo
  CC       ui/libcrypto_la-ui_util.lo
  CC       whrlpool/libcrypto_la-wp_dgst.lo
  CC       x509/libcrypto_la-by_dir.lo
  CC       x509/libcrypto_la-by_file.lo
  CC       x509/libcrypto_la-by_mem.lo
  CC       x509/libcrypto_la-x509_att.lo
  CC       x509/libcrypto_la-x509_cmp.lo
  CC       x509/libcrypto_la-x509_d2.lo
  CC       x509/libcrypto_la-x509_def.lo
  CC       x509/libcrypto_la-x509_err.lo
  CC       x509/libcrypto_la-x509_ext.lo
  CC       x509/libcrypto_la-x509_lu.lo
  CC       x509/libcrypto_la-x509_obj.lo
  CC       x509/libcrypto_la-x509_r2x.lo
  CC       x509/libcrypto_la-x509_req.lo
  CC       x509/libcrypto_la-x509_set.lo
  CC       x509/libcrypto_la-x509_trs.lo
  CC       x509/libcrypto_la-x509_txt.lo
  CC       x509/libcrypto_la-x509_v3.lo
  CC       x509/libcrypto_la-x509_vfy.lo
  CC       x509/libcrypto_la-x509_vpm.lo
  CC       x509/libcrypto_la-x509cset.lo
  CC       x509/libcrypto_la-x509name.lo
  CC       x509/libcrypto_la-x509rset.lo
  CC       x509/libcrypto_la-x509spki.lo
  CC       x509/libcrypto_la-x509type.lo
  CC       x509/libcrypto_la-x_all.lo
  CC       x509v3/libcrypto_la-pcy_cache.lo
  CC       x509v3/libcrypto_la-pcy_data.lo
  CC       x509v3/libcrypto_la-pcy_lib.lo
  CC       x509v3/libcrypto_la-pcy_map.lo
  CC       x509v3/libcrypto_la-pcy_node.lo
  CC       x509v3/libcrypto_la-pcy_tree.lo
  CC       x509v3/libcrypto_la-v3_akey.lo
  CC       x509v3/libcrypto_la-v3_akeya.lo
  CC       x509v3/libcrypto_la-v3_alt.lo
  CC       x509v3/libcrypto_la-v3_bcons.lo
  CC       x509v3/libcrypto_la-v3_bitst.lo
  CC       x509v3/libcrypto_la-v3_conf.lo
  CC       x509v3/libcrypto_la-v3_cpols.lo
  CC       x509v3/libcrypto_la-v3_crld.lo
  CC       x509v3/libcrypto_la-v3_enum.lo
  CC       x509v3/libcrypto_la-v3_extku.lo
  CC       x509v3/libcrypto_la-v3_genn.lo
  CC       x509v3/libcrypto_la-v3_ia5.lo
  CC       x509v3/libcrypto_la-v3_info.lo
  CC       x509v3/libcrypto_la-v3_int.lo
  CC       x509v3/libcrypto_la-v3_lib.lo
  CC       x509v3/libcrypto_la-v3_ncons.lo
  CC       x509v3/libcrypto_la-v3_ocsp.lo
  CC       x509v3/libcrypto_la-v3_pci.lo
  CC       x509v3/libcrypto_la-v3_pcia.lo
  CC       x509v3/libcrypto_la-v3_pcons.lo
  CC       x509v3/libcrypto_la-v3_pku.lo
  CC       x509v3/libcrypto_la-v3_pmaps.lo
  CC       x509v3/libcrypto_la-v3_prn.lo
  CC       x509v3/libcrypto_la-v3_purp.lo
  CC       x509v3/libcrypto_la-v3_skey.lo
  CC       x509v3/libcrypto_la-v3_sxnet.lo
  CC       x509v3/libcrypto_la-v3_utl.lo
  CC       x509v3/libcrypto_la-v3err.lo
  CC       compat/strndup.lo
  CC       compat/strnlen.lo
  CC       compat/strsep.lo
  CC       compat/inet_pton.lo
  CC       compat/timegm.lo
  CC       compat/reallocarray.lo
  CC       compat/recallocarray.lo
  CC       compat/timingsafe_memcmp.lo
  CC       compat/timingsafe_bcmp.lo
  CC       compat/arc4random.lo
  CC       compat/arc4random_uniform.lo
  CC       compat/getentropy_solaris.lo
  CCLD     libcompat.la
  CC       compat/libcompatnoopt_la-explicit_bzero.lo
  CCLD     libcompatnoopt.la
  CCLD     libcrypto.la
Undefined                       first referenced
 symbol                             in file
OPENSSL_ia32cap_P                   .libs/libcrypto.so.41.0.1.exp
ld: warning: symbol referencing errors
Making all in ssl
  CC       bio_ssl.lo
  CC       bs_ber.lo
  CC       bs_cbb.lo
  CC       bs_cbs.lo
  CC       d1_both.lo
  CC       d1_clnt.lo
  CC       d1_enc.lo
  CC       d1_lib.lo
  CC       d1_meth.lo
  CC       d1_pkt.lo
  CC       d1_srtp.lo
  CC       d1_srvr.lo
  CC       pqueue.lo
  CC       s3_cbc.lo
  CC       s3_lib.lo
  CC       ssl_algs.lo
  CC       ssl_asn1.lo
  CC       ssl_both.lo
  CC       ssl_cert.lo
ssl_cert.c: In function `SSL_add_dir_cert_subjects_to_stack':
ssl_cert.c:701: warning: implicit declaration of function `asprintf'
  CC       ssl_ciph.lo
ssl_ciph.c: In function `SSL_CIPHER_description':
ssl_ciph.c:1672: warning: implicit declaration of function `asprintf'
  CC       ssl_clnt.lo
  CC       ssl_err.lo
  CC       ssl_lib.lo
  CC       ssl_packet.lo
  CC       ssl_pkt.lo
  CC       ssl_rsa.lo
  CC       ssl_sess.lo
  CC       ssl_srvr.lo
  CC       ssl_stat.lo
  CC       ssl_txt.lo
  CC       ssl_versions.lo
  CC       t1_clnt.lo
  CC       t1_enc.lo
  CC       t1_hash.lo
  CC       t1_lib.lo
  CC       t1_meth.lo
  CC       t1_reneg.lo
  CC       t1_srvr.lo
  CCLD     libssl.la
Making all in tls
  CC       libtls_la-tls.lo
tls.c: In function `tls_error_vset':
tls.c:84: warning: implicit declaration of function `vasprintf'
tls.c:94: warning: implicit declaration of function `asprintf'
  CC       libtls_la-tls_client.lo
  CC       libtls_la-tls_bio_cb.lo
  CC       libtls_la-tls_config.lo
  CC       libtls_la-tls_conninfo.lo
tls_conninfo.c: In function `tls_get_peer_cert_hash':
tls_conninfo.c:76: warning: implicit declaration of function `asprintf'
  CC       libtls_la-tls_server.lo
  CC       libtls_la-tls_ocsp.lo
  CC       libtls_la-tls_peer.lo
  CC       libtls_la-tls_util.lo
tls_util.c: In function `tls_host_port':
tls_util.c:68: warning: implicit declaration of function `asprintf'
  CC       libtls_la-tls_verify.lo
  CCLD     libtls.la
Making all in include
Making all in openssl
Making all in apps
Making all in ocspcheck
  CC       http.o
http.c: In function `http_open':
http.c:345: warning: implicit declaration of function `asprintf'
  CC       ocspcheck.o
ocspcheck.c: In function `main':
ocspcheck.c:567: warning: long int format, size_t arg (arg 3)
ocspcheck.c:604: warning: long int format, size_t arg (arg 3)
  CC       compat/inet_ntop.o
  CC       compat/memmem.o
  CCLD     ocspcheck
Undefined                       first referenced
 symbol                             in file
OPENSSL_ia32cap_P                   /export/software/libressl-2.5.3/crypto/.libs/libcrypto.so
ld: fatal: symbol referencing errors. No output written to .libs/ocspcheck
collect2: ld returned 1 exit status
*** Error code 1
The following command caused the error:
echo "  CCLD    " ocspcheck;/bin/bash ../../libtool --silent --tag=CC    --mode=link gcc -std=gnu99  -g -O2 -Wall -std=gnu99 -fno-strict-aliasing    -o ocspcheck http.o ocspcheck.o  compat/inet_ntop.o compat/memmem.o /export/software/libressl-2.5.3/crypto/libcrypto.la  /export/software/libressl-2.5.3/ssl/libssl.la  /export/software/libressl-2.5.3/tls/libtls.la -lnsl -lsocket   -lresolv -ldl -lrt
make: Fatal error: Command failed for target `ocspcheck'
Current working directory /export/software/libressl-2.5.3/apps/ocspcheck
*** Error code 1
The following command caused the error:
fail=; \
if (target_option=k; case ${target_option-} in  ?) ;;  *) echo "am__make_running_with_option: internal error: invalid"  "target option '${target_option-}' specified" >&2;  exit 1;;  esac;  has_opt=no;  sane_makeflags=$MAKEFLAGS;  if {  if test -z '1'; then  false;  elif test -n ''; then  true;  elif test -n '' && test -n ''; then  true;  else  false;  fi;  }; then  sane_makeflags=$MFLAGS;  else  case $MAKEFLAGS in  *\\[\ \      ]*)  bs=\\;  sane_makeflags=`printf '%s\n' "$MAKEFLAGS"  | sed "s/$bs$bs[$bs $bs        ]*//g"`;;  esac;  fi;  skip_next=no;  strip_trailopt ()  {  flg=`printf '%s\n' "$flg" | sed "s/$1.*$//"`;  };  for flg in $sane_makeflags; do  test $skip_next = yes && { skip_next=no; continue; };  case $flg in  *=*|--*) continue;;  -*I) strip_trailopt 'I'; skip_next=yes;;  -*I?*) strip_trailopt 'I';;  -*O) strip_trailopt 'O'; skip_next=yes;;  -*O?*) strip_trailopt 'O';;  -*l) strip_trailopt 'l'; skip_next=yes;;  -*l?*) strip_trailopt 'l';;  -[dEDm]) skip_next=yes;;  -[JT]) skip_next=yes;;  esac;  case $flg in  *$target_option*) has_opt=yes; break;;  esac;  done;  test $has_opt = yes); then \
  failcom='fail=yes'; \
else \
  failcom='exit 1'; \
fi; \
dot_seen=no; \
target=`echo all-recursive | sed s/-recursive//`; \
case "all-recursive" in \
  distclean-* | maintainer-clean-*) list='ocspcheck openssl nc' ;; \
  *) list='ocspcheck openssl nc' ;; \
esac; \
for subdir in $list; do \
  echo "Making $target in $subdir"; \
  if test "$subdir" = "."; then \
    dot_seen=yes; \
    local_target="$target-am"; \
  else \
    local_target="$target"; \
  fi; \
  (CDPATH="${ZSH_VERSION+.}:" && cd $subdir && make  $local_target) \
  || eval $failcom; \
done; \
if test "$dot_seen" = "no"; then \
  make  "$target-am" || exit 1; \
fi; test -z "$fail"
make: Fatal error: Command failed for target `all-recursive'
Current working directory /export/software/libressl-2.5.3/apps
*** Error code 1
The following command caused the error:
fail=; \
if (target_option=k; case ${target_option-} in  ?) ;;  *) echo "am__make_running_with_option: internal error: invalid"  "target option '${target_option-}' specified" >&2;  exit 1;;  esac;  has_opt=no;  sane_makeflags=$MAKEFLAGS;  if {  if test -z ''; then  false;  elif test -n ''; then  true;  elif test -n '' && test -n ''; then  true;  else  false;  fi;  }; then  sane_makeflags=$MFLAGS;  else  case $MAKEFLAGS in  *\\[\ \       ]*)  bs=\\;  sane_makeflags=`printf '%s\n' "$MAKEFLAGS"  | sed "s/$bs$bs[$bs $bs        ]*//g"`;;  esac;  fi;  skip_next=no;  strip_trailopt ()  {  flg=`printf '%s\n' "$flg" | sed "s/$1.*$//"`;  };  for flg in $sane_makeflags; do  test $skip_next = yes && { skip_next=no; continue; };  case $flg in  *=*|--*) continue;;  -*I) strip_trailopt 'I'; skip_next=yes;;  -*I?*) strip_trailopt 'I';;  -*O) strip_trailopt 'O'; skip_next=yes;;  -*O?*) strip_trailopt 'O';;  -*l) strip_trailopt 'l'; skip_next=yes;;  -*l?*) strip_trailopt 'l';;  -[dEDm]) skip_next=yes;;  -[JT]) skip_next=yes;;  esac;  case $flg in  *$target_option*) has_opt=yes; break;;  esac;  done;  test $has_opt = yes); then \
  failcom='fail=yes'; \
else \
  failcom='exit 1'; \
fi; \
dot_seen=no; \
target=`echo all-recursive | sed s/-recursive//`; \
case "all-recursive" in \
  distclean-* | maintainer-clean-*) list='crypto ssl tls include apps tests man' ;; \
  *) list='crypto ssl tls include apps tests man' ;; \
esac; \
for subdir in $list; do \
  echo "Making $target in $subdir"; \
  if test "$subdir" = "."; then \
    dot_seen=yes; \
    local_target="$target-am"; \
  else \
    local_target="$target"; \
  fi; \
  (CDPATH="${ZSH_VERSION+.}:" && cd $subdir && make  $local_target) \
  || eval $failcom; \
done; \
if test "$dot_seen" = "no"; then \
  make  "$target-am" || exit 1; \
fi; test -z "$fail"
make: Fatal error: Command failed for target `all-recursive'

--- a/m4/check-libc.m4
+++ b/m4/check-libc.m4
@@ -150,6 +150,11 @@ fi
 ])
 
 AC_DEFUN([GENERATE_CRYPTO_PORTABLE_SYM], [
+AS_CASE([$host_cpu],
+	[i?86], [HOSTARCH=intel],
+	[x86_64], [HOSTARCH=intel],
+)
+AC_SUBST([HOSTARCH])
 crypto_sym=$srcdir/crypto/crypto.sym
 crypto_p_sym=./crypto/crypto_portable.sym
 echo "generating $crypto_p_sym ..."
@@ -206,6 +211,9 @@ if test "x$ac_cv_func_timingsafe_bcmp" = "xno" ; then
 fi
 if test "x$ac_cv_func_timingsafe_memcmp" = "xno" ; then
 	echo timingsafe_memcmp >> $crypto_p_sym
+fi
+if test "x$HOSTARCH" = "xintel" ; then
+	echo OPENSSL_ia32cap_P >> $crypto_p_sym
 fi
 if test "x$HOST_OS" = "xwin" ; then
 	echo posix_perror >> $crypto_p_sym

--- a/m4/check-libc.m4
+++ b/m4/check-libc.m4
@@ -153,6 +153,7 @@ AC_DEFUN([GENERATE_CRYPTO_PORTABLE_SYM], [
 AS_CASE([$host_cpu],
 	[i?86], [HOSTARCH=intel],
 	[x86_64], [HOSTARCH=intel],
+	[amd64], [HOSTARCH=intel],
 )
 AC_SUBST([HOSTARCH])
 crypto_sym=$srcdir/crypto/crypto.sym

